script.js

```diff
@@ -0,0 +1,6 @@
+const hambur = document.querySelector('.hamburger');
+const menu = document.querySelector('.menu-section');
+hambur.onclick = function myfunction() {
+  menu.classList.toggle('active');
+  hambur.classList.toggle('active');
+};
```

Comment:
- [ ] Please add the functionality such that when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears ☺️.

Reply:
Hi @AlphaNtihinduka 
sorry, I didn't understand could you show me an example for that